files/build_templates/sonic_debian_extension.j2

```diff
@@ -101,6 +101,10 @@ sudo mkdir -p $FILESYSTEM_ROOT_USR_LIB_SYSTEMD_SYSTEM
 sudo dpkg --root=$FILESYSTEM_ROOT -i $debs_path/ifupdown2_*.deb || \
     sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get -y install -f
 
+# Install a patched version of ipmitool  (and its dependencies via 'apt-get -y install -f')
+sudo dpkg --root=$FILESYSTEM_ROOT -i $debs_path/ipmitool_*.deb || \
+    sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get -y install -f
```

Comment:
In `build_debian.sh`, can you remove the `ipmitool` package installation from there, to make sure this version is the only version that gets installed?

Reply:
I pushed the changed asked above. 

root@sonic:/home/admin# show ver

SONiC Software Version: SONiC.master.0-dirty-20240209.233855
SONiC OS Version: 12
Distribution: Debian 12.4
Kernel: 6.1.0-11-2-amd64
Build commit: bd47fd155
Build date: Sat Feb 10 01:35:50 UTC 2024
Built by: commbuild@rajvm

Platform: x86_64-dellemc_s5212f_c3538-r0
HwSKU: DellEMC-S5212f-P-25G
ASIC: broadcom
ASIC Count: 1
Serial Number: CQY67C3
Model Number: 0VK93C
Hardware Revision: 01
Uptime: 07:59:58 up 4 min,  1 user,  load average: 5.00, 4.12, 1.78
Date: Tue 13 Feb 2024 07:59:58


root@sonic:/home/admin# ipmitool raw 0x04 0x2d 0x31
Get Device ID command failed: 0xb1 Unknown (0xB1)
Received a response with unexpected ID 0 vs. 1
 00 c0 01 80


root@sonic:/home/admin# dpkg -l ipmitool
||/ Name           Version      Architecture Description
+++-==============-============-============-==================================>
ii  ipmitool       1.8.19-4     amd64        utility for IPMI control with kern>